src/meta/src/backup_restore/backup_manager.rs

```diff
@@ -15,21 +15,22 @@
 use std::sync::Arc;
 
 use itertools::Itertools;
+use risingwave_backup::error::BackupError;
+use risingwave_backup::storage::BackupStorageRef;
+use risingwave_backup::{MetaBackupJobId, MetaSnapshotId};
 use risingwave_common::bail;
 use risingwave_hummock_sdk::HummockSstableId;
+use risingwave_pb::backup_service::BackupJobStatus;
 use tokio::task::JoinHandle;
 
-use crate::backup_restore::error::BackupError;
-use crate::backup_restore::meta_snapshot::MetaSnapshotBuilder;
-use crate::backup_restore::storage::BackupStorageRef;
-use crate::backup_restore::MetaSnapshotId;
+use crate::backup_restore::meta_snapshot_builder::MetaSnapshotBuilder;
 use crate::hummock::{HummockManagerRef, HummockVersionSafePoint};
 use crate::manager::{IdCategory, MetaSrvEnv};
 use crate::storage::MetaStore;
 use crate::MetaResult;
 
 pub enum BackupJobResult {
-    Finished,
+    Succeeded,
```

Comment:
We need to expose some metrics for backup and restore (e.g. success/fail rate, elasped time, ...)